templates/setup.cfg.tmpl

```diff
@@ -10,8 +10,42 @@ parentdir_prefix = {{PROJECT_NAME}}
 addopts = --doctest-modules --cov=./tests
 
 [flake8]
-exclude = __init__.py,versioneer.py,{{SRC}}/_version.py
-max-line-length = 120
+exclude = __init__.py,versioneer.py,src/_version.py
```

Comment:
In this line we must keep {{SRC}} instead of src

Reply:
Ops, sorry. Fixed